ansible/roles/debops.discourse/defaults/main.yml

```diff
@@ -0,0 +1,686 @@
+
+  # Discourse user account role
+  - name: '{{ discourse_database_user }}'
+    password: '{{ discourse_database_password }}'
+
+                                                                   # ]]]
+# .. envvar:: discourse__postgresql__dependent_databases [[[
+#
+# Configuration of PostgreSQL databases for the :ref:`debops.postgresql` Ansible
+# role.
+discourse__postgresql__dependent_databases:
+
+  - name:  '{{ discourse_database_name }}'
+    owner: '{{ discourse_database_name }}'
+
+                                                                   # ]]]
+# .. envvar:: discourse__postgresql__dependent_groups [[[
+#
+# Configuration of PostgreSQL groups for the :ref:`debops.postgresql` Ansible role.
+discourse__postgresql__dependent_groups:
+
+  - roles:  [ '{{ discourse_database_user }}' ]
+    groups: [ '{{ discourse_database_name }}' ]
+    database: '{{ discourse_database_name }}'
+
+                                                                   # ]]]
+# .. envvar:: discourse__postgresql__dependent_extensions [[[
+#
+# Configuration of PostgreSQL extensions for the :ref:`debops.postgresql` Ansible
+# role.
+discourse__postgresql__dependent_extensions:
+
+  - database: '{{ discourse_database_name }}'
+    extension: 'pg_trgm'
+  - database: '{{ discourse_database_name }}'
+    extension: 'hstore'
+
+                                                                   # ]]]
+# .. envvar:: discourse__postgresql__dependent_pgpass [[[
+#
+# The ``~/.pgpass`` configuration for :ref:`debops.postgresql` Ansible role.
+discourse__postgresql__dependent_pgpass:
+
+  - owner: '{{ discourse_user }}'
+    home: '{{ discourse_home }}'
+    database: '{{ discourse_database_name }}'
+    role: '{{ discourse_database_user }}'
+
+                                                                   # ]]]
+# .. envvar:: discourse__nginx_upstream_name [[[
+#
+# Name of the first :command:`nginx` upstream.
+discourse__nginx_upstream_name: 'discourse-{{discourse__instance_name}}'
+                                                                   # ]]]
+# .. envvar:: discourse__nginx__dependent_upstreams [[[
+#
+# List of :command:`nginx` upstreams for the :ref:`debops.nginx`
+# Ansible role.
+discourse__nginx__dependent_upstreams:
+
+  - name: '{{ discourse__nginx_upstream_name }}'
+    server: '{{ "unix:" + discourse_git_checkout + "/tmp/sockets/discourse.socket"
+             if discourse_unicorn_connection == "socket"
+             else "127.0.0.1:" + discourse_unicorn_port }}'
+
+                                                                   # ]]]
+# .. envvar:: discourse__nginx__dependent_servers [[[
+#
+# List of :command:`nginx` servers for the :ref:`debops.nginx` Ansible role.
+discourse__nginx__dependent_servers:
+
+  - '{{ discourse__nginx_server }}'
+
+                                                                   # ]]]
+# .. envvar:: discourse__nginx__dependent_maps [[[
+#
+#
+discourse__nginx__dependent_maps:
+
+  - name: thescheme
```

Comment:
Hmm, `nginx` maps are global, perhaps this should be namespaced just to be sure. `discourse_scheme`?